res/xml/qwertz_hu.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?> 
+<keyboard>
+  <row>
+    <key key0="q" key1="esc"    key2="0" key3="§" key4="\\" />
+    <key key0="w" key1="'"      key2="1" key4="|" />
+    <key key0="e" key1="&quot;" key2="2" key4="é" />
+    <key key0="r" key1="+"      key2="3" key4="^" />
+    <key key0="t" key1="!"      key2="4" key4="f11_placeholder"/>
+    <key key0="z" key1="%"      key2="5" key3="°" key4="f12_placeholder"/>
+    <key key0="u" key1="ű"      key2="6" key3="ü" key4="ú" />
+    <key key0="i" key1="="      key2="7" key3="`" key4="í" />
+    <key key0="o" key1="ő"      key2="8" key3="ö" key4="ó"/>
+    <key key0="p"               key2="9" key3="(" key4=")" />
```

Comment:
Some users of phone cases have a hard time to type characters too close to the edges of the screen (left and right). This is not a requirement but it could be appreciated, I should write this in the guidelines.

`(` and `)` could be on `k` and `§` on `s`. `esc` is now generally on `key4`, so `\` would have to be moved a bit far.